tools/clang/lib/Lex/HeaderSearch.cpp

```diff
@@ -601,6 +602,13 @@ const FileEntry *HeaderSearch::LookupFile(
     // If this was an #include_next "/absolute/file", fail.
     if (FromDir) return nullptr;
 
+    // HLSL Change - begin
+    // Normalize file name first
+    llvm::SmallString<128> NormalizedFileName = Filename;
+    llvm::sys::path::native(NormalizedFileName);
+    Filename = NormalizedFileName;
+    // HLSL Change - end
```

Comment:
I wasn't sure why this was needed, but saw that this is called with `FileName` containing slashes due to mapped files and such